src/experimental/ks-installer/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+description: kubesphere installer 
+name: ks-installer
+version: 2.1.1
```

Comment:
Both `appVersion` and `version` should be specified.

Reply:
I read document of Semantic Versioning, but The helm chart of ks-installer is based on the image version of ks-installer.So I set `version` to `v2.1.1` 。

Reply:
```
appVersion: 2.1.1 # the version of ks-installer itself
version: 0.1.0    # the version of this chart
```